Library/Homebrew/test/caveats_spec.rb

```diff
@@ -26,4 +26,39 @@ def caveats
       expect(described_class.new(f)).not_to be_empty
     end
   end
+
+  describe "#caveats" do
+    describe "when f.plist is not null" do
```

Comment:
The `describe "when ..."` blocks are redundant now. Everything else looks fine. 👍